lib/package/kinesis/compileMethodsToKinesis.js

```diff
@@ -93,6 +91,12 @@ module.exports = {
     }
   },
 
+  getIntegrationUri(http) {
+    return {
+      'Fn::Sub': 'arn:aws:apigateway:${AWS::Region}:kinesis:action/' + (http.action || 'PutRecord')
+    }
```

Comment:
Please use `${AWS::Partition}` instead of `aws` for consistency because of https://github.com/serverless-operations/serverless-apigateway-service-proxy/pull/97